README.md

```diff
@@ -4,19 +4,38 @@
 [![Build Status](https://travis-ci.org/Cloud-CV/Fabrik.svg?branch=master)](https://travis-ci.org/Cloud-CV/Fabrik)
 [![Coverage Status](https://coveralls.io/repos/github/Cloud-CV/Fabrik/badge.svg?branch=master)](https://coveralls.io/github/Cloud-CV/Fabrik?branch=master)
 
-Fabrik is an online collaborative platform to build, visualize and train deep learning models via a simple drag-and-drop interface. It allows researchers to collaboratively develop and debug models using a web GUI that supports importing, editing and exporting networks written in widely popular frameworks like Caffe, Keras, and TensorFlow.
+Fabrik is an online collaborative platform which allows one to build, visualize and train deep learning models via a simple drag-and-drop interface. It allows researchers to collectively develop and debug models using a web GUI which supports importing, editing and exporting networks written using popular frameworks like Caffe, Keras, and TensorFlow.
 
 <img src="/example/fabrik_demo.gif?raw=true">
 
-This app is presently under active development and we welcome contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
+This app is presently under active development and welcomes contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
 
 
 ## Installation Instructions
 
-Setting up Fabrik on your local machine is really easy. You can setup Fabrik using two methods:
+Setting up Fabrik on your local machine is very easy. You can setup Fabrik using two methods:
 
 ### Using Docker
```

Comment:
I don't think we need to add installation instructions for docker here, you can just add a reference from where anyone can install docker. Our README should be focused on setting up Fabrik. Can you remove this.

Reply:
yes I agree . I was asked to add It . will incorporate all the changes in the next commit . sorry
thanks!